src/backend/commands/player/kick/multi_kick.cpp

```diff
@@ -18,14 +18,12 @@ namespace big
 		virtual void execute(player_ptr player, const command_arguments& _args, const std::shared_ptr<command_context> ctx) override
 		{
 			if (g_player_service->get_self()->is_host())
-				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {}),
-				    dynamic_cast<player_command*>(command::get("hostkick"_J))->call(player, {});
+				dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {});
 
-			dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
-			script::get_current()->yield(700ms);
+			if (!g_player_service->get_self()->is_host() && player && player->is_valid() && player->is_host())
+				dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
 
-
-			if (player && !g_player_service->get_self()->is_host() && player->is_valid() && !player->is_host())
+			if (!g_player_service->get_self()->is_host() && player && player->is_valid() && !player->is_host())
```

Comment:
3/4th of the condition in these two if conditions are duplicates.

This whole function can be simplified to the following:
```cpp
if (g_player_service->get_self()->is_host())
{
	dynamic_cast<player_command*>(command::get("breakup"_J))->call(player, {});
}
else if (player && player->is_valid())
{
	if (player->is_host())
		dynamic_cast<player_command*>(command::get("oomkick"_J))->call(player, {});
	else
		dynamic_cast<player_command*>(command::get("desync"_J))->call(player, {});
}
```